### PR TITLE
update ES links

### PR DESCRIPTION
**Pull request by RafaelDavisH**
This PR is in replacement to the previous PR #40

### CODE REVIEW

**Comment by naomi-lgbt** [comment]
Hey @RafaelDavisH! 

Is this PR separate from #40?

**Comment by RafaelDavisH** [comment]
Yes. Disregard the last PR #40

**Review by moT01 — DISMISSED** [review]
I noticed some of the titles are still in English - `Git Clone` and `Python Main` among others. I'm guessing that's intentional though. If so, LGTM 🎉

File: locales/es.json [hunk 1]
```diff
@@ -29,35 +29,35 @@
...
-  "footer-col-3-3": "<a href='https://www.freecodecamp.org/news/pdf-password-remover-guide-how-to-remove-password-protection-from-a-pdf/'>PDF Password Remover</a>",
-  "footer-col-3-4": "<a href='https://www.freecodecamp.org/news/what-is-a-pdf-file-and-how-do-you-open-it-solved/'>What Is a PDF?</a>",
+  "footer-col-2-10": "<a href='https://www.freecodecamp.org/espanol/news/create-table-sql-server-con-ejemplo/'>Create Table SQL</a>",
+  "footer-col-3-1": "<a href='https://www.freecodecamp.org/espanol/news/diseno-web-responsive-como-hacer-que-un-sitio-web-se-vea-bien-en-telefonos-y-tabletas/'>Diseño Web Responsive</a>",
+  "footer-col-3-2": "<a href='https://www.freecodecamp.org/espanol/news/que-es-svg-explicacion-de-etiquetas-imagenes/'>¿Qué es un archivo SVG?</a>",
+  "footer-col-3-3": "<a href='https://www.freecodecamp.org/espanol/news/que-es-un-archivo-pdf-como-lo-abres/'>PDF Password Remover</a>",
+  "footer-col-3-4": "<a href='https://www.freecodecamp.org/news/what-is-a-pdf-file-and-how-do-you-open-it-solved/'>¿Qué es un Archivo PDF?</a>",
```
Comment (naomi-lgbt) [inline locales/es.json]: Should this link point to an `espanol` article?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: locales/es.json (hunk 1 above)
Comment (RafaelDavisH) [reply locales/es.json]: Yes 🤦🏽

**Comment by RafaelDavisH** [comment]
> I noticed some of the titles are still in English - `Git Clone` and `Python Main` among others. I'm guessing that's intentional though. If so, LGTM 🎉 

Yes, by convention we’re not translating some of the titles and others because those articles are not translated yet.

**Review by scissorsneedfoodtoo — APPROVED** [review]
Tested locally and everything LGTM :+1: :+1: 

Thanks for making all these changes, @RafaelDavisH, and thanks everyone else for your thorough review.

**Pull request merged**